### PR TITLE
Release 2.3.0.

### PR DESCRIPTION
**Pull request by joshuajbouw**
### Added

- A precompile which exposes NEAR's random number generator was added by [@mfornet] as requested by 
[@birchmd]. ([#368] [#297])
- London hard fork support was added by [@birchmd]. ([#244])

### Changes

- The gas limit for `deposit` and `ft_on_transfer` were changed as they were not attaching enough
gas, as changed by [@mrLSD]. ([#389])

### Fixes

- There was an issue with the original storage not actually being stored. Unfortunately, previous 
transactions can't be updated with this change. This has been fixed by [@birchmd]. ([#390])
- Call arguments were intended to have a value attached to them to make it equivalent to an ETH
call. This was fixed in a backwards compatible manner by [@andrcmdr], as reported by [@birchmd].
([#351] [#309])
- A refund now happens in the case of an error on the `ExitToNear` precompile by [@birchmd]. 
([#311])

### Removed

- Betanet support was dropped and will no longer be supported by [@joshuajbouw]. ([#388])

[#390]: https://github.com/aurora-is-near/aurora-engine/pull/390
[#389]: https://github.com/aurora-is-near/aurora-engine/pull/389
[#388]: https://github.com/aurora-is-near/aurora-engine/pull/388
[#368]: https://github.com/aurora-is-near/aurora-engine/pull/368
[#351]: https://github.com/aurora-is-near/aurora-engine/pull/351
[#311]: https://github.com/aurora-is-near/aurora-engine/pull/311 
[#309]: https://github.com/aurora-is-near/aurora-engine/issues/309
[#297]: https://github.com/aurora-is-near/aurora-engine/issues/297
[#244]: https://github.com/aurora-is-near/aurora-engine/pull/244

### CODE REVIEW

**Review by mrLSD — APPROVED** [review]

File: CHANGES.md [hunk 1]
```diff
@@ -7,6 +7,43 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+gas, as changed by [@mrLSD]. ([#389])
+
+### Fixes
+
+- There was an issue with the original storage not actually being stored. Unfortunately, previous 
+transactions can't be updated with this change. This has been fixed by [@birchmd]. ([#390])
+- Call arguments were intended to have a value attached to them to make it equivalent to an ETH
+call. This was fixed in a backwards compatible manner by [@andrcmdr], as reported by [@birchmd].
+([#351] [#309])
+- A refund now happens in the case of an error on the `ExitToNear` precompile by [@birchmd].
```
Comment (birchmd) [inline CHANGES.md:30]: This was merged behind a feature flag, so it's not actually enabled. Still waiting on @sept-en / @mfornet to do the necessary migration before we can enable this.

Comment (joshuajbouw) [reply CHANGES.md:30]: Will update after the call.

Comment (joshuajbouw) [reply CHANGES.md]: Fixed

File: CHANGES.md [hunk 2]
```diff
@@ -7,6 +7,41 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+### Changes
+
+- The gas limit for `deposit` and `ft_on_transfer` were changed as they were not attaching enough
+gas, as changed by [@mrLSD]. ([#389])
+
+### Fixes
+
+- There was an issue with the original storage not actually being stored. Unfortunately, previous 
+transactions can't be updated with this change. This has been fixed by [@birchmd]. ([#390])
```
Comment (artob) [inline CHANGES.md:26]: This is a pretty poor description that will not be comprehensible to most anybody who reads this changelog. What is "the original storage"? Cool, old transactions can't be updated--what is the actual impact?

**Pull request merged**